### DISCUSSION TITLE
Infinity reroll issue

### DISCUSSION THREAD

**Discussion by mirakhany** [isid0re/SoloLeveling#307] [Q&A]
Hi. finally I got runes for infinity and my sorceress made one.
But she rerolled immediately.
I think this is big. People can lost two Ber rune.
Plz check this issue.

**Comment by isid0re** [comment]
Thanks I will definately check it out. 

A couple questions?
1. what were the stats on the infinity before it rerolled?
2. And did you help it in any way to male the infinity?

**Reply by mirakhany** [reply]
1. I couldn't touch infinity because rerolled immediately. so I don't know about stats. sry.
2. No. I just watched.

I'm not expert but is this 13 problem?
Config.KeepRunewords.push("[type] == polearm # [convictionaura] >= 13");

**Reply by mirakhany** [reply]
and 1 more issue.
My assassin(Leveling build) made Call To Arms and I checked "kept Call To Arms" message from BS manager.
But when I joined assassin manually, I couldn't find any Call To Arms.(inventory, stash, swap weapon)
I guess she sold or dropped Call To Arms.
Plz check this issue.
-added-
My Necromancer also "Made runeword: Heart of the Oak Flail", "Runeword Kept Heart of the Ork Flail" message from BS manager.
But no Heart of the Oak item in game.

**Reply by isid0re** [reply]
The infinity issue has been fixed. The keep line had a typo of convictionaura 13 when it should have been 12

Im.checking the others. Thank you.
